tests/testthat/helper-scope.R

```diff
@@ -37,20 +32,18 @@ renv_tests_scope <- function(packages = character(),
   # use temporary library
   lib <- tempfile("renv-library-")
   ensure_directory(lib)
-  libpaths <- .libPaths()
-  .libPaths(lib)
+  renv_scope_libpaths(lib, envir = envir)
 
-  defer(envir = envir, {
-    setwd(owd)
-    unlink(lib, recursive = TRUE)
-    .libPaths(libpaths)
-  })
+  # make sure we clean up the library path when we're done too
+  defer(unlink(lib, recursive = TRUE), envir = envir)
```

Comment:
You can eliminate this by using `renv_scope_tempfile()` instead of `tempfile()` above.

Reply:
Thanks!